WholeGenomeGermlineSingleSample.wdl

```diff
@@ -39,23 +39,21 @@ version 1.0
 #import "../../../../structs/dna_seq/germline/GermlineStructs.wdl"
 
 # Git URL import
-import "https://raw.githubusercontent.com/gatk-workflows/five-dollar-genome-analysis-pipeline/1.2.0/tasks/UnmappedBamToAlignedBam.wdl" as ToBam
-import "https://raw.githubusercontent.com/gatk-workflows/five-dollar-genome-analysis-pipeline/1.2.0/tasks/AggregatedBamQC.wdl" as AggregatedQC
-import "https://raw.githubusercontent.com/gatk-workflows/five-dollar-genome-analysis-pipeline/1.2.0/tasks/GermlineVariantDiscovery.wdl" as Calling
-import "https://raw.githubusercontent.com/gatk-workflows/five-dollar-genome-analysis-pipeline/1.2.0/tasks/Qc.wdl" as QC
-import "https://raw.githubusercontent.com/gatk-workflows/five-dollar-genome-analysis-pipeline/1.2.0/tasks/Utilities.wdl" as Utils
-import "https://raw.githubusercontent.com/gatk-workflows/five-dollar-genome-analysis-pipeline/1.2.0/tasks/BamToCram.wdl" as ToCram
-import "https://raw.githubusercontent.com/gatk-workflows/five-dollar-genome-analysis-pipeline/1.2.0/tasks/VariantCalling.wdl" as ToGvcf
-import "https://raw.githubusercontent.com/gatk-workflows/five-dollar-genome-analysis-pipeline/1.2.0/structs/GermlineStructs.wdl"
+import "https://raw.githubusercontent.com/genome/five-dollar-genome-analysis-pipeline/anvil-wgs-qc/tasks/AggregatedBamQC.wdl" as AggregatedQC
+import "https://raw.githubusercontent.com/genome/five-dollar-genome-analysis-pipeline/anvil-wgs-qc/tasks/Qc.wdl" as QC
+import "https://raw.githubusercontent.com/genome/five-dollar-genome-analysis-pipeline/anvil-wgs-qc/structs/GermlineStructs.wdl"
 
 # WORKFLOW DEFINITION
-workflow WholeGenomeGermlineSingleSample {
+workflow WholeGenomeSingleSampleQc {
   input {
+    File input_bam
+    File input_bam_index
     SampleAndUnmappedBams sample_and_unmapped_bams
```

Comment:
we definitely won't have unmapped BAMs: 
https://github.com/genome/five-dollar-genome-analysis-pipeline/blob/935f55154a992c3b85bc6acc26433dcbf6501bc0/structs/GermlineStructs.wdl#L6

Reply:
In fact, I wonder it the only input really needed is an output file prefix. That should be the only part of the struct that's used in the QC.